data/saas/config/rollbar_config.yml

```diff
@@ -0,0 +1,91 @@
+saas_config:
+  - fides_key: rollbar_connector
+    name: Rollbar SaaS Config
+    type: rollbar
+    description: A sample schema representing the Rollbar connector for Fidesops
+    version: 0.0.1
+
+    connector_params:
+      - name: domain
+      - name: read_access_token
+      - name: write_access_token
+      - name: page_size
```

Comment:
We're not using this `page_size` param in the config, we should make use of it or remove it.